streaming/src/main/scala/org/apache/spark/streaming/Checkpoint.scala

```diff
@@ -41,6 +41,13 @@ class Checkpoint(ssc: StreamingContext, val checkpointTime: Time)
   val checkpointDuration = ssc.checkpointDuration
   val pendingTimes = ssc.scheduler.getPendingTimes().toArray
   val sparkConfPairs = ssc.conf.getAll
+  @transient val recoverableAccuNameToId = ssc.recoverableAccuNameToId
```

Comment:
I may not describe it clearly, sorry. It used to be inside of `val trackedAccs = ...`, but it caused serialization issue: it try to serializate StreamingContext when doing checkpoint. So I move it out of `trackedAccs ` definition explicitly to break the dependency of `ssc`.